src/trace/implementations/ord_neu.rs

```diff
@@ -720,7 +719,6 @@ mod key_batch {
     ///
     /// The `L` parameter captures how the updates should be laid out, and `C` determines which
     /// merge batcher to select.
-    #[derive(Abomonation)]
```

Comment:
Same as above; removing a derived `Abomonation` without introducing `Serialize, Deserialize`. Idk the consequences.